src/backend/utils/parse-modules.ts

```diff
@@ -0,0 +1,153 @@
+import {classDecorators, componentMetadata} from '../../tree/decorators';
+import {functionName} from '../../utils/function-name';
+
+export const AUGURY_TOKEN_ID_METADATA_KEY = '__augury_token_id';
+
+declare const ng;
+
+const resolveNgModuleDecoratorConfig = (m) => {
+  if (m.decorators) {
+    return m.decorators.reduce((prev, curr, idx, decorators) =>
+      prev ? prev : decorators[idx].type.prototype.toString() === '@NgModule' ? decorators[idx].args[0] : null, null);
+  }
+
+  return Reflect.getMetadata('annotations', m).find(decorator => decorator.toString() === '@NgModule');
+};
+
+export const parseModules = (firstRootDebugElement: any): {[key: string]: any} => {
+  // TODO(steven.kampen): This uses a private API. Can it be improved?
+  const bootstrappedModule = firstRootDebugElement.injector.get(ng.coreTokens.ApplicationRef)._injector.instance;
```

Comment:
@clbond I've been talking to Igor from the core team about it, and there may be a safer way to do this but I'm waiting on full details.